azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -15,9 +15,41 @@
  */
 package azkaban.execapp;
 
+import static azkaban.Constants.EventReporterConstants.ATTEMPT_ID;
```

Comment:
just import the whole EventReporterConstants, this is not pretty.

Reply:
imported static class EventReporterConstants